src/app.js

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env node
+'use strict';
+let fs = require('fs');
+let path = require('path');
+const chalk = require('chalk');
+const fetch = require('node-fetch');
+
+let uri = process.argv[2]
+let lengthProcess = process.argv.length
+let stats = process.argv[3];
+
+
+const controller = (arrLinks) => {
+    if (lengthProcess === 3) {
+        arrLinks.forEach(link => {
+            console.log(chalk`{yellow ${uri} ${link.slice(0, 50)}}`);
+        })
+    } if (stats === '--stats' || stats === '--s') {
+        statsBasic(arrLinks)
+    } if (stats === '--validate' || stats === '--val') {
+        verifyLinks(arrLinks)
+
+    }
+}
+
+
+const fileRoute = () => {
+    new Promise((resolve, reject) => {
+        if (path.isAbsolute(uri)) {
+            let directory = fs.readdirSync(uri, 'utf-8')
+            let arrDir = directory.forEach(re => {
+                let extUri = path.extname(re)
```

Comment:
Ser más especifica en el nombre de la variable
```suggestion
                let extencionFolder = path.extname(re)
```